site/src/components/pagination.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Pagination
+description: Pagination related components in the Atlas Design System
+template: standard
+---
+
+# Pagination
+
+The pagination component consists of several elements:
+
+- `pagination-previous` and `pagination-next` for incremental navigation
+- `pagination-list` which displays page items for navigation to individual pages
+- `pagination-link` for page numbers and `pagination-ellipsis` for range numbers
+
+All elements are optional so you can compose your pagination as you wish.
+
+```html
+<nav class="pagination" role="navigation" aria-label="pagination">
+	<a class="pagination-previous" aria-label="previous">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-left"></span>
```

Comment:
Docons won't work on the Atlas site. Can you try inline svg? I pulled these from Docons

Arrow left - <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448"><path d="M448 224H62.625l148.688 148.688-22.625 22.625L1.375 208 188.688 20.688l22.625 22.625L62.625 192H448v32z"/></svg>
Arrow right - <svg xmlns="http://www.w3.org/2000/svg" viewBox="-255 57 448 448"><path d="M-253.625 281v-32H131.75L-16.938 100.313 5.687 77.688 193 265 5.687 452.313l-22.625-22.625L131.75 281h-385.375z"/></svg>